03-terraform-outputs-across-stacks/stacks/VPC/stack.tm.hcl

```diff
@@ -0,0 +1,8 @@
+stack {
+  name        = "vpc"
+  description = "vpc"
+  id          = "c2c67b6a-5cc5-4129-b684-f471548781d5"
+  before      = [
+        "/stacks/ec2"
```

Comment:
@vtimd run `terramate experimental run-order` - this will fail because `/stacks/ec2` doesn't exist. It needs to be `stacks/EC2` as the stack is named using uppercase

Reply:
```
vtimd@Tims-Air 03-terraform-outputs-across-stacks % terramate experimental run-order
2024-01-02T14:15:39-06:00 WRN path /03-terraform-outputs-across-stacks/stacks/EC2 not found in configuration action=root.StacksByPath basedir=/03-terraform-outputs-across-stacks/stacks/vpc paths=["../EC2"]
/03-terraform-outputs-across-stacks/stacks/ec2
/03-terraform-outputs-across-stacks/stacks/vpc
vtimd@Tims-Air 03-terraform-outputs-across-stacks % 
```

Reply:
```
vtimd@Tims-Air 03-terraform-outputs-across-stacks % terramate experimental run-order

/03-terraform-outputs-across-stacks/stacks/vpc
/03-terraform-outputs-across-stacks/stacks/ec2
vtimd@Tims-Air 03-terraform-outputs-across-stacks % 
```

Reply:
Updated before path. Tested. Pushed.